cmd/integration_link.go

```diff
@@ -141,38 +141,42 @@ List of available integrations:
 				autoDeploy := c.Bool("auto-deploy")
 				noAutoDeploy := c.Bool("no-auto-deploy")
 				if autoDeploy && noAutoDeploy {
-					errorQuit(errors.New("cannot define both auto-deploy and no-auto-deploy"))
+					errorQuitWithHelpMessage(errors.New("cannot define both auto-deploy and no-auto-deploy"), c, "integration-link-create")
 				}
 
 				deployReviewApps := c.Bool("deploy-review-apps")
 				noDeployReviewApps := c.Bool("no-deploy-review-apps")
 				if deployReviewApps && noDeployReviewApps {
-					errorQuit(errors.New("cannot define both deploy-review-apps and no-deploy-review-apps"))
+					errorQuitWithHelpMessage(errors.New("cannot define both deploy-review-apps and no-deploy-review-apps"), c, "integration-link-create")
 				}
 
 				destroyOnClose := c.Bool("destroy-on-close")
 				noDestroyOnClose := c.Bool("no-destroy-on-close")
 				if destroyOnClose && noDestroyOnClose {
-					errorQuit(errors.New("cannot define both destroy-on-close and no-destroy-on-close"))
+					errorQuitWithHelpMessage(errors.New("cannot define both destroy-on-close and no-destroy-on-close"), c, "integration-link-create")
 				}
 				hoursBeforeDestroyOnClose := c.Uint("hours-before-destroy-on-close")
 
 				destroyOnStale := c.Bool("destroy-on-stale")
 				noDestroyOnStale := c.Bool("no-destroy-on-stale")
 				if destroyOnStale && noDestroyOnStale {
-					errorQuit(errors.New("cannot define both destroy-on-stale and no-destroy-on-stale"))
+					errorQuitWithHelpMessage(errors.New("cannot define both destroy-on-stale and no-destroy-on-stale"), c, "integration-link-create")
 				}
 				hoursBeforeDestroyOnStale := c.Uint("hours-before-destroy-on-stale")
 
 				allowReviewAppsFromForks := c.Bool("allow-review-apps-from-forks")
 				noAllowReviewAppsFromForks := c.Bool("no-allow-review-apps-from-forks")
 
 				if allowReviewAppsFromForks && noAllowReviewAppsFromForks {
-					errorQuit(errors.New("cannot define both allow-review-apps-from-forks and no-allow-review-apps-from-forks"))
+					errorQuitWithHelpMessage(errors.New("cannot define both allow-review-apps-from-forks and no-allow-review-apps-from-forks"), c, "integration-link-create")
 				}
 
 				awareOfSecurityRisks := c.Bool("aware-of-security-risks")
 
+				if awareOfSecurityRisks && !c.IsSet("allow-review-apps-from-forks") {
+					errorQuitWithHelpMessage(errors.New("flag --aware-of-security-risks is useless without allowing review apps from forks"), c, "integration-link-create")
```

Comment:
This message differs from what is requested [here](https://github.com/Scalingo/project-items/issues/68#issuecomment-1489906089). Note I haven't checked the respective times at which each (the comment, the PR) was posted.

Reply:
Yup, Benjamin's review changed over time, now its review is `flag --aware-of-security-risks must be used in conjunction with --aware-of-security-risks`, changing this now!